benchmark.py

```
@@ -0,0 +1,826 @@
+
+import aiohttp
+import asyncio
+import time
+import random
+import argparse
+import subprocess
+import shutil
+import os
+from pathlib import Path
+import statistics
+from typing import AsyncGenerator, AsyncIterator, Any, NamedTuple
```

Ruff (0.12.2)

Import from `collections.abc` instead: `AsyncGenerator`, `AsyncIterator`

Import from `collections.abc`

(UP035)